src/data/downloads.tsx

```diff
@@ -54,15 +54,33 @@ export const Downloads: Array<Download> = [
         name: 'Install Instructions',
         details: (
           <>
+            <pre>
+              <code>
+                {`curl https://repo.jellyfin.org/install-debuntu.sh | sudo bash`}
+              </code>
+            </pre>
+            <p>
+              If you don't have <code>curl</code> installed, you can try <code>wget</code> instead:
+            </p>
+            <pre>
+              <code>
+                {`wget -O- https://repo.jellyfin.org/install-debuntu.sh | sudo bash`}
+              </code>
+            </pre>
+            <p>
+              For more advanced users, the full steps are:
+            </p>
             <pre>
               <code>
                 {`sudo apt install curl gnupg
 sudo mkdir /etc/apt/keyrings
-curl -fsSL https://repo.jellyfin.org/$( awk -F'=' '/^ID=/{ print $NF }' /etc/os-release )/jellyfin_team.gpg.key | sudo gpg --dearmor -o /etc/apt/keyrings/jellyfin.gpg
+DISTRO="$( awk -F'=' '/^ID=/{ print $NF }' /etc/os-release )"
+CODENAME="$( awk -F'=' '/^VERSION_CODENAME=/{ print $NF }' /etc/os-release )"
+curl -fsSL https://repo.jellyfin.org/\${DISTRO}/jellyfin_team.gpg.key | sudo gpg --dearmor -o /etc/apt/keyrings/jellyfin.gpg
 cat <<EOF | sudo tee /etc/apt/sources.list.d/jellyfin.sources
 Types: deb
-URIs: https://repo.jellyfin.org/$( awk -F'=' '/^ID=/{ print $NF }' /etc/os-release )
-Suites: $( awk -F'=' '/^VERSION_CODENAME=/{ print $NF }' /etc/os-release )
+URIs: https://repo.jellyfin.org/\${DISTRO}
+Suites: \${CODENAME}
```

Comment:
I don't know for sure if this is the correct escape here or if it even matters, so worth checking.

Reply:
For future benefit: yes, this was correct :-)